### ISSUE TITLE
How to make VirtualizedScrollPane scroll to the bottom?

### ISSUE THREAD

**Issue by PashaTurok** [FXMisc/RichTextFX#1100]
I use RichTextFX 0.10.9 and I have the following code:

```
    InlineCssTextArea textArea = new InlineCssTextArea();
    VirtualizedScrollPane scrollPane = new VirtualizedScrollPane(textArea);
       .....
    Platform.runLater(() -> {
           .....
           scrollPane.scrollYToPixel(Double.MAX_VALUE);//LINE X
    });
```
At line X I need to scroll textArea to bottom. But it doesn't work. Please, could anyone tell me how to do it?

**Comment by PashaTurok** [comment]
Could anyone say if this is a bug or I do something wrong?

**Comment by Jugen** [comment]
Hi there are various ways of accomplishing this, depending on what it is you want to achieve, try:
```java
textArea.requestFollowCaret();
```
or maybe:
```java
int last = textArea.getParagraphs().size()-1;
textArea.showParagraphAtBottom( last );
```
and then there's also `textArea.showParagraphAtTop( p );`

**Comment by PashaTurok** [comment]
@Jugen Thank you very much. This variant `textArea.requestFollowCaret();` works fine.

**Comment by PashaTurok** [comment]
@Jugen  When I try to compile this code:
```
int last = textArea.getParagraphs().size() - 1;
```

I get compilation error: 
```
java.util.List.size() is defined in an inaccessible class or interface
```

I use JDK 11 and JPMS. Is this problem with my code? Does it work for you?

**Comment by Jugen** [comment]
Using JDK 11 with class path works for me, can't say about modules ?

**Comment by PashaTurok** [comment]
Could you give me the following output:
```
System.out.println(textArea.getParagraphs().getClass().getCanonicalName());
```

**Comment by PashaTurok** [comment]
@Jugen I just wander what is wrong with this class or one of its interfaces. Maybe I miss some modules in module-info ...

**Comment by Jugen** [comment]
Requested output is: `org.reactfx.collection.SuspendableListWrapper`

Are you using the fat jar: `richtextfx-fat-0.10.9.jar` ?

**Comment by PashaTurok** [comment]
@Jugen No, I don't use richtextfx-fat. I solved the problem with the compilation error by adding `requires reactfx;` to my module-info. Thank you for your help.
